src/Utils/APIUtils.ts

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
-const api_endPoint = "https://api.climatehealers.com/"
-// const api_endPoint = "https://foodhealers.climatehealers.com/"
+// const api_endPoint = "https://api.climatehealers.com/"
+const api_endPoint = "https://foodhealers.climatehealers.com/"
```

Comment:
@Shoebmalik007 , why is this url used is this prod url? are we deploying for prod?

Reply:
It was just for checking that everything is working properly  in prod.